ambient_toolbox/middleware/current_request.py

```diff
@@ -0,0 +1,25 @@
+from threading import local
+
+_threadlocal = local()
```

Comment:
Seit Python 3.7 gibt es das contextvars Modul: [Docs](https://docs.python.org/3/library/contextvars.html)

Würde man die hier einsetzen, wäre diese Middleware sogar async-safe. Vllt. eine Überlegung wert?

Weitere Links:
https://valarmorghulis.io/tech/201904-contextvars-and-thread-local/
https://stackoverflow.com/questions/68856006/is-there-any-reason-to-use-python-threading-local-rather-than-a-contextvar-in

Reply:
Sounds nice! @michael-agel-ambient what do you think?

Reply:
@fbinz Did you have a look at the code in general aka did you do a review?

Reply:
@fbinz Great idea! Changed this in 40ddc0e98c665f5fc25d30c21dc5d2469ca71e3d, along with some type annotations.

Reply:
Thx for your effort! I'll have a look at it next week. 👌